Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "ISC"
 name = "epd-waveshare"
 readme = "README.md"
 repository = "https://github.com/Caemor/epd-waveshare.git"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Do you need it released soon on crates.io? 
Else I would most likely wait for the embedded-hal 1.0 release, then fix/include a few of the nearly finished prs and also release a 1.0 version. But I can always release another version inbetween if needed earlier.

Reply:
Nope, take your time.